doc/nrf/templates/Ecosystem_integration_template_README.rst

```diff
@@ -0,0 +1,135 @@
+.. _Ecosystem_integration:
+
+Template: Ecosystem integration
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   Provide a short introductory description of the tool or service (third-party), indicating the value it brings to |NCS|.
+   Refer to the following instructions to create the description:
+
+   * Give the title that starts with the tool or service name with "integration" as a suffix, for example, "Memfault integration".
+   * Place the documentation inside the :file:`nrf/doc/nrf` folder.
+   * Add the file name in the toctree of the :file:`ext_comps` RST file.
```

Comment:
```suggestion
   * Add the file name in the ToC tree of the :file:`ext_comps` RST file.
```

Reply:
Updated as ``.. toctree:`` to be consistent with other documents - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/mesh_model_main_README.html#bt-mesh-model-main-template